src/node/main.ts

```diff
@@ -23,28 +23,38 @@ export const shouldSpawnCliProcess = (args: UserProvidedArgs): boolean => {
   )
 }
 
+export interface OpenCommandPipeArgs {
```

Comment:
docs: can we add a description to this? Not sure what this is for without digging further

Reply:
Like I assume some "Open Command" but no idea what this is actually used for

Reply:
Will do.

Reply:
Added a comment.  It would be cool if we could import this type directly from Code without having to import the code itself which will mess with the `out` directory structure.  I wonder if we can do this with `import type` but I have not tested it yet.  For now we can duplicate the types I suppose.

Reply:
Actually I am going to put a link to the type on the Code side instead.